client/src/components/NavBar/NavBar.tsx

```diff
@@ -101,16 +99,15 @@ export function Navbar(props: any) {
         ></MenuButton>
       </ButtonWrapper>
 
-        <Link to="/">Home</Link>
-        <Link to="advanced-search">Advanced Search</Link>
-        <Link to="results">Results</Link>
-        <Link to="review">Review</Link>
-        <Link to="reviews">Reviews</Link>
-        <StyledButton onClick={() => testAPICalls()}>Test API</StyledButton>
-        <Link to={"/single-result/ChIJn58N1B9gUocRpAXOXPbFcOo"}>
-          Single Result
-        </Link>
-      </details>
```

Comment:
Removing details element removes the dropdown and styles that @MatteoThomas added. Where would the dummy data you added be shown? Is there a link for it in the nav?

Reply:
I removed this because it was missing the beginning tag, it was thowing errors that prevented me from running my code. I wasn't sure where the beginning tag went so I just removed it. It seems like this happened during a merge conflict. It runs fine without so I just did that and created a ticket for @MatteoThomas to fix it later:
https://github.com/Healthy-Plate-Project/Healthy-Eating-App/issues/144

Reply:
Yeah, it's fine with me. The details element was a temporary solution for better navigation design.

Reply:
Okay thanks for the update.